.github/workflows/unittests.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         path: go/src/github.com/cilium/hubble
     - uses: actions/setup-go@v1
       with:
-        go-version: '1.14.7'
+        go-version: '1.15'
```

Comment:
does this imply `1.15.0`?

Reply:
Good point. I assumed that this was following the Go version scheme where `1.15` means `1.15.0`. To make sure, I changed it to `1.15.0` and go a failure for the Github Action so then reverted back to `1.15`.